docs/public/specifications/tests/RequirementsAndNormalizationOfDomainNames.md

```diff
@@ -117,7 +117,7 @@ can be returned to the user.
 
 Message Tag                    | Level    | Arguments    | Message ID for message tag
 :------------------------------|:---------|:-------------|:----------------------------------------------------------
-AMBIGUOUS_DOWNCASING           | CRITICAL | unicode_name | Ambiguous downcaseing of character "{unicode_name}" in the domain name. Use all lower case instead.
+AMBIGUOUS_DOWNCASING           | CRITICAL | unicode_name | Ambiguous downcasing of character "{unicode_name}" in the domain name. Use all lower case instead.
```

Comment:
This will require editing a msgid in Zonemaster::Engine::Normalization::Error.

Reply:
> This will require editing a msgid in Zonemaster::Engine::Normalization::Error.

The accompanying edit is already done in <https://github.com/zonemaster/zonemaster-engine/pull/1366>.